tests/yam/agama/agama.pm

```diff
@@ -14,20 +14,23 @@ use testapi qw(
   assert_screen
   get_required_var
   enter_cmd
+  select_console
 );
 
 sub run {
     my $self = shift;
     my $test = get_required_var('AGAMA_TEST');
 
-    assert_script_run("playwright test --trace on --project chromium --config /usr/share/e2e-agama-playwright tests/" . $test . ".spec.ts", timeout => 1200);
-    $self->upload_traces();
-    enter_cmd 'reboot';
+    assert_script_run("agama-integration-tests /usr/share/agama/integration-tests/build/" . $test . ".js", timeout => 1200);
+    upload_traces();
+    select_console 'installation';
+    save_screenshot();
```

Comment:
if you want the screenshot of the mocha output better put it before `upload_traces` otherwise if log recovery fails we might not get the screenshot.